### ISSUE TITLE
Configurable logging

### ISSUE THREAD

**Issue by ciryx** [kishikawakatsumi/UICKeyChainStore#87]
I'm using this library in a banking app and it would be great if there was a way to turn of the NSLogs for release builds. Reasoning behind this is that we want to make sure that we don't log sensitive information.

**Comment by kishikawakatsumi** [comment]
Indeed. Thanks for pointing that out. I'm planning such mechanism for logging.
